packages/phone-number-privacy/signer/test/integration/domain.test.ts

```diff
@@ -476,16 +486,21 @@ describe('domainService', () => {
           now: res1.body.status.now,
         },
       })
+      const eval1 = thresholdPoprfClient.unblindPartialResponse(
+        Buffer.from(res1.body.signature, 'base64')
+      )
+      expect(eval1.toString('base64')).toEqual(expectedEval)
 
       // submit identical request with nonce set to 1
-      const req2 = (await signatureRequest(undefined, 1))[0]
+      req1.options.nonce = defined(1)
```

Comment:
Oh weird...I guess this should've been failing with 401 before (or if we remove the nonce update)? Do you know why it was passing?

Reply:
Yeah! So before, the test case for the repeated requests was not only testing the nonce change -- we were creating a separate request with an entirely new blinded message + session, so while this was a repeated request for a single account, it didn't really test the exact same request overall (since the message, session, and nonce were all different). The previous nonce failure test case (below in the diff, I can tag you down there!) was failing because of the signature over the request not being correct -- we updated the nonce but didn't delete the old signature (the `NoString` part), re-sign the message, and include this signature in the new request.

Reply:
Ahh makes sense! Great catch